docs/data/date-pickers/adapters-locale/adapters-locale.md

```diff
@@ -240,8 +240,8 @@ This prop is available on all pickers.
 ### Custom day of week format
 
 Use `dayOfWeekFormatter` to customize day names in the calendar header.
-This prop takes the short name of the day provided by the date library as an input, and returns it's formatted version.
-The default formatter only keeps the first letter and capitalises it.
+This prop takes the date of the day provided by the date library as an input, and returns a formatted string.
+The default formatter only keeps the first letter of the name and capitalises it.
```

Comment:
Maybe add a small warning card saying that it also takes the string as it's 1st param but that this one will be removed in v7?